brave/brave-core
# [Android] add on demand reader mode menu item

Resolves brave/brave-browser/issues/29626



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Enable SpeedReader and go to any page mentioned in https://github.com/brave/brave-browser/issues/23418.
2. After the page finishes loading, check there is an "View in reader mode" item in menu.
3. Click on "View in reader mode"  to make current page reload in SpeedReader mode.
